files/fr/_redirects.txt

```diff
@@ -4295,7 +4295,6 @@
 /fr/docs/Web/CSS/Média_paginés	/fr/docs/Web/CSS/Paged_Media
 /fr/docs/Web/CSS/Outils	/fr/docs/conflicting/Web/CSS_c2c099599c0a58c69d1390033045f244
 /fr/docs/Web/CSS/Outils/Générateur_de_courbe_de_Bézier	/fr/docs/orphaned/Web/CSS/Tools/Cubic_Bezier_Generator
-/fr/docs/Web/CSS/Outils/Générateur_de_dégradés_linéaires	/fr/docs/Web/CSS/Tools/Linear-gradient_Generator
```

Comment:
This file was almost empty (be it with french content) and as useless as it was, I went with deleting it.